xml/src/main/java/org/ehcache/xml/XmlConfiguration.java

```diff
@@ -364,18 +366,35 @@ public URL getURL() {
   /**
    * Creates a new {@link CacheConfigurationBuilder} seeded with the cache-template configuration
    * by the given {@code name} in the XML configuration parsed using {@link #parseConfiguration()}
```

Comment:
The lead sentence should end with a period (http://www.oracle.com/technetwork/articles/java/index-137868.html#firstsentence).